integration/test-integration/src/test/resources/testng-cc-with-apim.xml

```diff
@@ -28,18 +28,28 @@
         <listener class-name="org.wso2.choreo.connect.tests.listener.TestNGListener"/>
     </listeners>
 
-    <test name="apim-startup-and-cc-startup" parallel="false">
+    <test name="Start API-M and prepare API-M clients" parallel="false">
         <classes>
             <!-- To avoid apim restarting everytime while writing tests, follow the instructions in integration/test-integration/readme.md -->
```

Comment:
can you update the readme file with the execution flow with when and where to add new test cases with small examples. Also update the readme file name as README.md

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2624/commits/8f3da3bc19b02e60e2946ddaf269f381f33e527d